payloads/v8/_interactions/autocomplete.ts

```diff
@@ -1,6 +1,6 @@
-import type { APIApplicationCommandInteractionData, APIBaseInteraction, InteractionType } from '../index';
+import type { APIBaseInteraction, APIChatInputApplicationCommandInteractionData, InteractionType } from '../index';
 
 export type APIApplicationCommandAutocompleteInteraction = APIBaseInteraction<
 	InteractionType.ApplicationCommandAutocomplete,
-	APIApplicationCommandInteractionData
+	APIChatInputApplicationCommandInteractionData
```

Comment:
This doesn't feel like the right type for this..

Reply:
I might be reading this wrong, but isn't this correct? 
https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-interaction-data-structure

Reply:
Just checked, it's fine